flower_shop/__manifest__.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "flower_shop",
+
+    'summary': """
+        Short (1 phrase/line) summary of the module's purpose, used as
+        subtitle on modules listing or apps.openerp.com""",
+
+    'description': """
+        Long description of module's purpose
+    """,
+
+    'author': "My Company",
+    'website': "https://www.yourcompany.com",
+
+    # Categories can be used to filter modules in modules listing
+    # Check https://github.com/odoo/odoo/blob/16.0/odoo/addons/base/data/ir_module_category_data.xml
+    # for the full list
+    'category': 'Uncategorized',
+    'version': '0.1',
+
+    # any module necessary for this one to work correctly
+    'depends': ['base','sale','stock','website'],
```

Comment:
missing dependencies, this might work on your testing DB. However the reason this is important is that you want to make sure whenever the  module is installed the required modules are installed with it

```suggestion
    'depends': ["website_sale", "stock", "base_geolocalize"],
```